meson_options.txt

```diff
@@ -0,0 +1,2 @@
+option('tests', type : 'boolean', value : true,
```

Comment:
Note that as a boolean, the dependencies that specify this for required will still be successfully found if available.

You can also/instead use a `type: 'feature', value: 'auto'`, which can be force disabled, force enabled, or left as a default "check if it's possible, use it if it is".

https://mesonbuild.com/Build-options.html#features